common/makefile.mk

```diff
@@ -22,10 +22,19 @@ JUJU_TARBALL_URL=https://launchpad.net/$(JUJU_PROJECT)/$(JUJU_MAJOR_MINOR)/$(JUJ
 
 .PHONY: build
 build: $(JUJU_TARBALL) $(JUJU_PATCH)
+
 	rm -rf $(JUJU_SRC)  # Go doesn't play nice with existing files.
+
+	# Extract the original tarball, apply the fake-juju patch and create
+	# synlinks in the original tree for the additional fake-juju sources.
```

Comment:
"syMlinks" and I believe "the" in "the additional fake-juju sources" is not needed